recipe/meta.yaml

```
@@ -9,34 +8,42 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: d41fd2650bb4221fae5a61f9c2767158f9727c4aaca95e24da86394feb704220
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-#  noarch: python
+  # numba not available for s390x
+  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - poetry
+    - {{ compiler('c') }}
+    - poetry-core >=1.0.0
     - cffi
     - setuptools
-    - {{ compiler('c') }}
   host:
+    - python
     - pip
-    - python 3
     - poetry
   run:
-    - python 3
-    - numpy
+    - python
+    - numpy {{numpy}}
```

```suggestion
    - numpy >=1.18,<2
```
Numpy is not used during the build, so there is no need to pin to a strict version.